.editorconfig

```diff
@@ -11,9 +11,13 @@ trim_trailing_whitespace = true
 indent_style = tab
 indent_size = 4
 
-[{.babelrc,.eslintrc,.rtlcssrc,*.json,*.yml}]
+[*.{json,yml,feature}]
 indent_style = space
 indent_size = 2
 
 [*.md]
 trim_trailing_whitespace = false
+
+[{.babelrc,.eslintrc,.rtlcssrc}]
```

Comment:
Curious why this is separated from the above block that has the same properties?

Reply:
The last time I checked it was not working as expected with the same block but now it is.

I have reverted these changes. Also, I have removed the `.babelrc` and `.eslintrc` files since we are not using them anymore.